src/libraries/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -34,6 +34,22 @@ public static IEnumerable<object[]> Names_TestData()
             yield return new object[] { "\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B", "\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B\uD83D\uDC3B" };
         }
 
+        public static IEnumerable<object[]> Names_TestDataQuoted()
+        {
+            yield return new object[] { " name ", "\" name \"" };
```

Comment:
Did you miss the case of escaping a backslash?

Reply:
Included